### DISCUSSION TITLE
Choix de couleur en hexadécimal impossible

### DISCUSSION THREAD

**Discussion by pleco27** [JeromeDevome/GRR#239] [Q&A]
Bonjour,
Dans le menu: Administration/Type de réservation/Modification d'un type de réservation:
Impossible de choisir la couleur de fond en hexadécimal:
L'anomalie réside dans le fait que je ne peux ni effacer ni remplacer directement la zone "Choix de la couleur de fond en hexadécimal"
J'ai essayé sous Firefox et Chrome, sous Windows(10) et Ubuntu(22.04): rien à faire
Merci
pleco27

**Comment by JeromeDevome** [comment]
Bonjour,
Normalement vous pouvez sélectionner via un sélecteur de couleur.
Il est impossible de saisir directement dans le champ en Hexa.

**Reply by pleco27** [reply]
Bonsoir Jerome,
Oui, je peux sélectionner la plupart des couleurs avec le sélecteur graphique
Mais, par exemple, le magenta ne passe pas au sélecteur (#FF00FF)
Ceci dit, ce n'est pas bloquant: une couleur approchée, accessible sur le mode graphique, fait l'affaire...
pleco27
